src/test/java/pl/stormit/tinyurl/controller/UrlControllerTest.java

```diff
@@ -69,9 +69,9 @@ void shouldReturnStatusCreatedWhenCreateShortUrlCorrectly() throws Exception {
     @Test
     void shouldThrowAnExceptionWhenLongUrlIsNull() throws Exception {
         //given
-        urlDto = new UrlDto(null, "");
+        urlDto = new UrlDto(null, "", null);
```

Comment:
Maybe it's worth to add a new constuctor with some default values.

Reply:
I will correct this in next task - refactoring